spec.bs

```diff
@@ -3045,9 +3087,19 @@ An interest group ad is a [=struct=] with the following [=struct/items=]:
 : <dfn>metadata</dfn>
 :: Null or a [=string=]. Extra arbitary information about this ad, passed to `generateBid()`.
 : <dfn>buyer reporting ID</dfn>
-:: Null or a [=string=]. Will be passed in place of interest group name to `reportWin()`, subject to k-anonymity checks.
+:: Null or a [=string=]. Will be passed in place of interest group name to `reportWin()`, subject to

```

Comment:
Can we link to `reportWin()`'s actual definition here?

Reply:
done. There are many of these in the spec and I'll link them to their definitions in a cleanup PR.

Reply:
Nice, thanks, I think that'd be nice, but no rush (low prio).